lib/new_relixir/instrumenters/phoenix.ex

```diff
@@ -0,0 +1,25 @@
+defmodule NewRelixir.Instrumenters.Phoenix do
+  @moduledoc """
+  New Relic instrumenter for Phoenix controllers.
+
+  It relies on the instrumentation API provided by `Phoenix.Endpoint`. To set it up,
+  include this module in the list of `instrumenters` of your Endpoint config:
+
+      config :my_app, MyAppWeb.Endpoint,
+        instrumenters: [NewRelixir.Instrumenters.Phoenix],
+
+  """
+  alias NewRelixir.Utils
+
+  def phoenix_controller_call(:start, _compile_metadata, %{conn: conn}) do
+    if NewRelixir.active?, do: Utils.transaction_name(conn)
```

Comment:
All these `active?` calls don't feel quite good to look at, but since that's something that can change during runtime, I think there's not much we can do for now.